test/kubernetes/e2e/defaults/testdata/sleep_server.yaml

```diff
@@ -0,0 +1,117 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: sleep-server
+---
+# This is a test server that sleeps based on the x-sleep-time-ms header and returns a 200
+apiVersion: v1
+kind: Pod
+metadata:
+  name: sleep-server
+  namespace: sleep-server
+  labels:
+    app: sleep-server
+    version: v1
+    app.kubernetes.io/name: sleep-server
+spec:
+  terminationGracePeriodSeconds: 1  # Wait only 1 seconds instead of default 30
+  containers:
+    - name: sleep-server
+      image: python:latest
```

Comment:
Note: reconsider this... was thinking this would save use having to download a 50-60MB echo server that support sleep, but this image is way bigger.

Reply:
changed to ~15MB alpine image